docs/samples/v1beta1/torchserve/config.properties

```diff
@@ -1,10 +1,11 @@
 inference_address=http://0.0.0.0:8085
-management_address=http://0.0.0.0:8081
+management_address=http://0.0.0.0:8085
```

Comment:
Same as the inference endpoint port?

Reply:
Yes, Torchserve has added the flexibility of using same port for both inference and management.